.buildkite/pipeline-upload.sh

```diff
@@ -15,12 +15,17 @@ if [[ -n $BUILDKITE_TAG ]]; then
     "https://github.com/solana-labs/solana/releases/$BUILDKITE_TAG"
   buildkite-agent pipeline upload ci/buildkite-release.yml
 else
-  buildkite-agent pipeline upload ci/buildkite.yml
-fi
+  if [[ $BUILDKITE_BRANCH =~ ^pull ]]; then
+    # Add helpful link back to the corresponding Github Pull Request
+    buildkite-agent annotate --style info --context pr-backlink \
+      "Github Pull Request: https://github.com/solana-labs/solana/$BUILDKITE_BRANCH"
+  fi
 
-if [[ $BUILDKITE_BRANCH =~ ^pull ]]; then
-  # Add helpful link back to the corresponding Github Pull Request
-  buildkite-agent annotate --style info --context pr-backlink \
-    "Github Pull Request: https://github.com/solana-labs/solana/$BUILDKITE_BRANCH"
-fi
+  if [[ $BUILDKITE_MESSAGE =~ GitBook: ]]; then
```

Comment:
Where is `$BUILDKITE_MESSAGE` getting set?

Reply:
Magically by buildkite